src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx

```diff
@@ -140,6 +140,11 @@ export class StemEligibilityView extends React.Component {
       <div className={divClassName}>
         <fieldset className="schemaform-field-template schemaform-first-field">
           <legend className={legendClassName}>
+            <div className="sr-only">
+              {this.renderIsEdithNourseRogersScholarshipCheck()}
```

Comment:
Each of these methods render a `li` element they should be wrapped in a `ul`

Reply:
added <ul>